lua/lspconfig/ui/lspinfo.lua

```diff
@@ -51,6 +51,7 @@ local cmd_type = {
 
 local function make_config_info(config, bufnr)
   local config_info = {}
+  local default_config = require(string.format('lspconfig.server_configurations.%s', config.name)).default_config
```

Comment:
This will break for custom server configurations, will need to revise before merge